t/plugin/ext-plugin/extra-info.t

```diff
@@ -181,3 +182,156 @@ GET /hello
 --- error_code: 503
 --- error_log
 failed to receive RPC_HTTP_REQ_CALL: closed
+
+
+
+=== TEST 5: ask response body (not exist)
```

Comment:
Let's add a test with empty response body

Reply:
It doesn't make sense to add a test with an empty response body in the `ext-plugin-pre-req` plugin. I guess you want me to add this test case in the `ext-plugin-post-resp` plugin.
However, an empty response body usually occurs when the upstream returns a 304 status code or the request method is HEAD, which needs to be handled separately in subsequent PRs. I have added `TODO`.